cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1379,7 +1379,7 @@ declare namespace Deno {
   export type KvMutation =
     & { key: KvKey }
     & (
-      | { type: "set"; value: unknown }
+      | { type: "set"; value: unknown; expiration?: Date }
```

Comment:
In line with @igorzi's comments, I would prefer we change this to be `expireIn?: number`.